### DISCUSSION TITLE
How incorporate time-varying covariates into PopED

### DISCUSSION THREAD

**Discussion by Vincent813** [andrewhooker/PopED#66] [Q&A]
Dear all,

I would like to introduce a time dependence on the Favail parameter (called FRel here). I have written it in the ode function but I was not able to use it in the ff_fun fonction.

I have this kind of error when running _plot_model_prediction()_ function : 
_Error in data.frame(var = c("A1"), time = times_dose, value = c(DOSE * : 
object 'FRel' not found_

Any suggestion ?
Best regards,
Vincent

**Comment by andrewhooker** [comment]
Hi Vincent,

Do you think you could send an example that I can run on my side? 

Best regards,
Andy

**Reply by Vincent813** [reply]
Hi Andy,

Did you receive my email?

Best regards,
Vincent
